logstash/forwarder.sls

```diff
@@ -0,0 +1,20 @@
+{%- set config_file = salt['pillar.get']('logstash:forwarder:config_file') %}
+{%- set certificate = salt['pillar.get']('logstash:forwarder:certificate') %}
+{%- set environment = salt['pillar.get']('logstash:forwarder:environment', 'production') %}
+
+logstash-forwarder:
+  pkg.installed:
+    - refresh: True
+  service.running:
+    - enable: True
+
+/etc/logstash-forwarder.crt:
+  file.managed:
+    - source: salt://{{certificate}}
```

Comment:
Je kan het beste de prefix niet meenemen in de source om extra flexibel te kunnen zijn (en aangezien andere formulas het ook zo doen)